packages/airnode-utilities/src/evm/gas-prices/types.ts

```diff
@@ -35,3 +35,27 @@ export interface FetchOptions {
   provider: ethers.providers.Provider;
   chainOptions: ChainOptions;
 }
+
+export interface LatestBlockPercentileGasPriceStrategy {
+  gasPriceStrategy: 'latestBlockPercentileGasPrice';
+  percentile: number;
+  minTransactionCount: number;
+  pastToCompareInBlocks: number;
+  maxDeviationMultiplier: number;
+}
+export interface ProviderRecommendedGasPriceStrategy {
+  gasPriceStrategy: 'providerRecommendedGasPrice';
+  recommendedGasPriceMultiplier: number;
+}
+
+export interface ConstantGasPriceStrategy {
+  gasPriceStrategy: 'constantGasPrice';
+  gasPrice: PriorityFee;
+}
+
+export type GasPriceOracleStrategy =
+  | LatestBlockPercentileGasPriceStrategy
+  | ProviderRecommendedGasPriceStrategy
+  | ConstantGasPriceStrategy;
+
+export type GasPriceOracleConfig = GasPriceOracleStrategy[];
```

Comment:
Isn't this pretty much just a copy&paste of the config types? Can't we use those instead?

Reply:
I don't see why not (I saw the old gas prices had the types separately so I did it the same way), I'll make the change :smile:

Reply:
So as it turns out `airnode-validator` uses` airnode-utilities` and has it as a `tsconfig.json` reference and adding `airnode-validator` to `airnode-utilities` causes a circular dependency error so this may not be possible without changing our `tsconfig` settings. 

But I don't think we'd want to do that as circular dependencies are not recommended, right?

Reply:
We most certainly don't want a circular dependency :sweat_smile: It looks like `airnode-validator` has `airnode-utilities` as a dependency but doesn't use it anywhere. I think you can remove it from `airnode-validator` and that way you can use `airnode-validator` in `airnode-utilities`.